app/views/releases/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  <h1>Release</h1>
+  <hr/>
+  <h3><%= @release.title %> <i> (<%= @release.released_on.strftime('%B %d, %Y') %>) </i></h3>
+  <h5><%= @release.subtitle %></h5>
+  <p><%= @release.body %></p>
+  <%= link_to 'Back', releases_path, :class => "btn btn-primary btn-sm" %>
```

Comment:
It would be nice to have an edit button here. That way if you edit a release, get redirected to this page to view it and notice a mistake you can click to edit.

Reply:
<img width="1280" alt="Screen Shot 2021-12-21 at 3 57 31 PM" src="https://user-images.githubusercontent.com/81119399/147013123-620311e6-ba70-41bd-8604-0af444e0045f.png">

Let me know if the button size and color are ok.
Thanks.